Frameworks/UIKit/UIControl.mm

```diff
@@ -420,7 +428,26 @@ - (void)touchesMoved:(NSSet*)touchSet withEvent:(UIEvent*)event {
         return;
     }
 
-    [self sendActionsForControlEvents:UIControlEventTouchDragInside];
+    if ([touchSet containsObject:_controlEventTouch]) {
+        CGPoint point = [_controlEventTouch locationInView:self];
+        BOOL currentTouchInside = [self pointInside:point withEvent:event];
+
+        if (currentTouchInside != _touchInside) {
+            _touchInside = currentTouchInside;
+            if (currentTouchInside) {
+                [self sendActionsForControlEvents:UIControlEventTouchDragEnter];
+            } else {
+                [self sendActionsForControlEvents:UIControlEventTouchDragExit];
+            }
+        }
+
+        // TODO: Investigate fat-fingers
```

Comment:
it means we may be a bit too precise on our drag in/out (considering a slack value for a touch point vs. a single mouse point), but I think this is fine for now

---
In reply to: [96942924](https://github.com/Microsoft/WinObjC/pull/1731#discussion_r96942924) [](ancestors = 96942924)